NAnt.Extensions/Tasks/Microsoft/SignTool.cs

```diff
@@ -80,13 +80,14 @@ public class SignTask : Task
 
         [BuildElement("items")]
         public FileSet Items { get; set; }
+        public static string DefaultSignToolPath = @"C:\Program Files (x86)\Microsoft SDKs\Windows\v7.0A\Bin";
```

Comment:
This should be an internal variable or leave it alone.  This doesn't help with fixing the issue you are trying to resolve.